### ISSUE TITLE
errata for 2.1 change log

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag#4915]
I notice a few typos in the 2.1 [change log](https://www.w3.org/TR/2025/REC-WCAG21-20250506/#changelog).  The errors do _not_ exist in 2.2 nor the [2.1 errata](https://www.w3.org/WAI/WCAG21/errata/) page.

Per the [repo readme](https://github.com/w3c/wcag?tab=readme-ov-file#version-specific-documentation) I should be looking for a branch or folder named `WCAG-2.1` but I have not had any luck with that.

@kfranqueiro please hit me with a clue bat as to what file to make the corrections.  Thank you!

**Comment by bruce-usab** [comment]
Changes since the W3C Recommendation of 05 June 2018:

- In the Status of This Document the paragraph beginning "This document has been reviewed by W3C Members..." appears twice. The first instance of this paragraph has been removed.
- In the Introduction, several (but not all) references to "WCAG 2.0" **<del>has</del> <ins>have</ins>** been changed to "WCAG 2.1".
- In the 0.5.2 Numbering in WCAG 2.1, the words "critera" and "ccriteria" have been changed to "criteria".
- In 1.4.10 Reflow, the first note had a supernumary "Note" indicator which has been removed.
- In 1.4.13 Content on Hover or Focus, the word "dismissable" has been changed to "dismissible".
- In 4. Robust, repetition of the word "by" has been removed.
- In 5.2.2 Full pages, the third note began with "New" which has been removed.
- In 5.3.1 Required Components of a Conformance Claim the editorial note "In WCAG 2.0 this was a dated URI, which may need to be adjusted when this becomes a Rec." has been removed.
- In the definition for keyboard interface, the second (of three) **note<ins>s</ins>** has been changed to be an example of the first note, leaving only two actual notes.
- In the definition for technology, the third note has been changed to be an example.
- In 7. Input Purposes for User Interface Components, the word "county" has been changed to "country".
- In 1.3.4 Orientation, the note referencing "binary display orientation" has been clarified to read "content is not necessarily restricted to landscape or portrait display orientation".
- In a note in the definition of accessibility supported, references to "Conformance Criterion" were changed to "Conformance Requirement".
- In the definition of relative luminance, the red threshold was updated from 0.03928 to 0.04045.
- In 4.1.1 Parsing one note was deleted, and two notes were added, including: "This Success Criterion should be considered as always satisfied for any content using HTML or XML."

**Comment by kfranqueiro** [comment]
Yes, there is a branch named `WCAG-2.1`. Here is a [pointer to that section of the change log on that branch](https://github.com/w3c/wcag/blob/WCAG-2.1/guidelines/index.html#L753).

If you are editing locally, `git checkout WCAG-2.1` before creating your own branch to edit from.

Is there only one typo annotated in what you pasted in your last comment, or am I missing more?

**Comment by bruce-usab** [comment]
Just two typos.  Turns out searching for branch names are case sensitive.

**Comment by patrickhlauke** [comment]
Will be closed by https://github.com/w3c/wcag/pull/4919 (for some reason, GH isn't showing the relationship here)
